metabrainz/templates/admin/stats/top-tokens.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'admin/master.html' %}
+{% block body %}
+  <h1>Top Tokens</h1>
+
+    <h2>Non commercial users</h2>
+    <p>
+       Users in good standing are not listed here.
+    </p>
+    {{ user_list(non_commercial) }}
+
+    <h2>Commercial users</h2>
+    {{ user_list(commercial) }}
+
+{% endblock %}
+
+{% macro user_list(users) -%}
```

Comment:
Could we reuse the macro from the other template in the previous PR?

Reply:
We can't; there is one columns less in this view.